src/app/containers/Timestamp/index.jsx

```diff
@@ -80,10 +44,19 @@ const TimestampContainer = ({ lastPublished, firstPublished }) => {
     return null;
   }
 
+  const publishedDatetime = formatUnixTimestamp(firstPublished, longNumeric);
+  const updatedDatetime = formatUnixTimestamp(lastPublished, longNumeric);
+
   return (
     <GridItemConstrainedMedium>
-      {defaultTimestamp(firstPublished)}
-      {updatedTimestamp(lastPublished, firstPublished)}
+      <Timestamp datetime={publishedDatetime}>
+        {humanReadable(firstPublished)}
+      </Timestamp>
+      {lastPublished !== firstPublished ? (
+        <Timestamp datetime={updatedDatetime}>
+          Updated {humanReadable(lastPublished)}
```

Comment:
Acknowledging that this is hardcoded to the English translation for `Updated` as we will be implementing a internationalisation library when include other services/languages